go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/ElrondNetwork/elrond-go-core v1.1.19
 	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
 	github.com/ElrondNetwork/elrond-go-logger v1.0.7
-	github.com/ElrondNetwork/elrond-vm-common v1.3.14
+	github.com/ElrondNetwork/elrond-vm-common v1.3.15-0.20220824133059-36f123566431
```

Comment:
why did we needed the new vm-common? Should put here a proper tag.
Can you reference here the new PRs needed in the vm-common?

Reply:
the new vm-common was related to the deleteMetaData, but not specifically tot he multiple allowed addresses:
https://github.com/ElrondNetwork/elrond-vm-common/pull/126
and was added directly to this PR in order to test the deleteMetaData functionality together

Reply:
great :+1: 
Will wait for the proper tag commit.

Reply:
undid the vm-common upgrade and integration test fix because it seems the fix is not complete.